.github/ISSUE_TEMPLATE/01_support_new_library.yml

```
@@ -0,0 +1,71 @@
+name: "📚 Support a new library"
+description: Submit a request to add support for a third-party library
+title: "Support for {maven_coordinates}"
+labels:
+  - library-new-request
+assignees:
+  - kimeta
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the following details to help us evaluate and prioritize your request.
+
+  - type: input
+    id: maven_coordinates
+    attributes:
+      label: Full Maven coordinates
+      description: Include a version that you need. The system might support older versions so more people can benefit.
+      placeholder: groupId:artifactId:version
+    validations:
+      required: true
+
+  - type: input
+    id: source_code_link
+    attributes:
+      label: Library source code URL
+      description: Link to the upstream source repository (e.g., GitHub).
+      placeholder: https://github.com/org/repo
+    validations:
+      required: true
+
+  - type: input
+    id: test_suite_link
+    attributes:
+      label: Link to the existing test suite
+      description: Provide a link to the library's test suite (enter N/A if it does not exist).
+      placeholder: for example, https://github.com/netty/netty/tree/4.2/testsuite-common (or N/A if the suite can't be found)
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: self_support
+    attributes:
+      label: Contribution preference
+      description: Check this if you want to [contribute](blob/master/CONTRIBUTING.md) the reachability metadata yourself instead of using the AI generation.
```

```suggestion
      description: Check this if you want to [contribute](https://github.com/oracle/graalvm-reachability-metadata/blob/master/CONTRIBUTING.md) the reachability metadata yourself instead of using the AI generation.
```
Link wasn't found on testing